### PR TITLE
chore(mojaloop/#3458): nodejs upgrade

### PR DESCRIPTION
**Pull request by oderayi**
chore(mojaloop/#3458): NodeJS version upgrade - https://github.com/mojaloop/project/issues/3458
- Upgraded nodejs from v16 to v18 LTS
- Applied CI changes as previously implemented in [sdk-scheme-adapter](https://github.com/mojaloop/sdk-scheme-adapter/pull/453)
- Updated NPM dependencies
- Resolved all audit issues
- Migrated `master` branch to `main` branch

### CODE REVIEW

File: test/unit/lib/api-routes/config.test.js [hunk 1]
```diff
@@ -112,7 +112,8 @@ describe('API route /config', () => {
       }
       const res = await request(app).put(`/api/config/user`).send(newConfig)
       expect(res.statusCode).toEqual(422)
-      expect(res.body.errors).toStrictEqual([{"location": "body", "msg": "Invalid value", "param": "CALLBACK_ENDPOINT", "value": ""}])
+      console.log(res.body.errors)
+      expect(res.body.errors).toStrictEqual([{"location": "body", "msg": "Invalid value", "path": "CALLBACK_ENDPOINT", "type": "field", "value": ""}])
```
Comment (oderayi) [inline test/unit/lib/api-routes/config.test.js:116]: Verify that ttk-ui is updated to support this updated response format.

Comment (oderayi) [inline test/unit/lib/api-routes/config.test.js:116]: Verify that ttk-ui is updated to support this updated response format.

File: package.json [hunk 2]
```diff
@@ -50,86 +50,87 @@
...
     "@hapi/boom": "10.0.1",
     "@hapi/good": "9.0.1",
-    "@hapi/hapi": "^21.3.1",
+    "@hapi/hapi": "^21.3.2",
     "@hapi/inert": "7.1.0",
-    "@hapi/vision": "7.0.1",
-    "@mojaloop/central-services-logger": "^11.2.0",
+    "@hapi/vision": "7.0.3",
+    "@mojaloop/central-services-logger": "^11.2.2",
     "@mojaloop/ml-testing-toolkit-shared-lib": "^13.0.0",
```
Comment (oderayi) [inline package.json:69]: Upgrade ttk shared lib

File: package.json [hunk 3]
```diff
@@ -50,86 +50,87 @@
...
+    "@mojaloop/ml-testing-toolkit-shared-lib": "^14.0.0",
+    "@mojaloop/sdk-standard-components": "17.1.3",
     "@now-ims/hapi-now-auth": "2.1.0",
     "@types/socket.io": "^3.0.1",
     "adm-zip": "^0.5.10",
     "atob": "2.1.2",
-    "axios": "^0.27.2",
-    "chai": "^4.3.7",
+    "axios": "^1.5.0",
+    "chai": "^4.3.8",
```
Comment (vijayg10) [inline package.json:76]: Can we get rid of carrot symbol (^) for all the dependencies as we are following for other repositories?

**Review by vijayg10 — APPROVED** [review]

**Pull request merged**